npm-admin-client/src/market_helpers.ts

```diff
@@ -158,6 +158,37 @@ export async function findEscrowPda(
   return response.body;
 }
 
+/**
+ * For the provided market publicKey, return the commission queue account PDA (publicKey) for that market.
+ *
+ * @param program {program} anchor program initialized by the consuming client
+ * @param marketPk {PublicKey} publicKey of a market
+ * @returns {FindPdaResponse} PDA of the payment queue
+ *
+ * @example
+ *
+ * const marketPk = new PublicKey('7o1PXyYZtBBDFZf9cEhHopn2C9R4G6GaPwFAxaNWM33D')
+ * const commissionQueuePda = await findEscrowPda(program, marketPK)
+ */
+export async function findMarketCommissionQueuePda(
```

Comment:
Nitpick: we use `commission_payments_queue` within the protocol codebase. Should this be a little more like that?

Reply:
Was just trying to keep it a bit more concise, but I can update to be more consistent

Reply:
It's a nitpick, either way is fine, though usually I'd err on the side of consistency.